components/private/Math/src/MDCMath.h

```diff
@@ -94,3 +94,20 @@ static inline CGFloat MDCSqrt(CGFloat value) {
   return sqrtf(value);
 #endif
 }
+
+/**
+ Expand `rect' to the smallest rect containing it with integral origin and
+ size.
```

Comment:
Not really true. Can you please document the `scale` functionality? I'm assuming that this is meant to round to pixel boundaries?